src/runtime_src/core/edge/user/shim.cpp

```diff
@@ -715,6 +715,39 @@ xclSKGetCmd(xclSKCmd *cmd)
   return ret;
 }
 
+int
+shim::
+xclAIEGetCmd(xclAIECmd *cmd)
+{
+  int ret;
```

Comment:
Combine with line 725.  In C++ it is best to keep definitions where they are used when possible.  Same goes a few other places.

Reply:
done